### PR TITLE
Adding new data retrieveing nd model predictor files in dashboard

### PR DESCRIPTION
**Pull request by anurag0511**
Added the model predictors and some test files for the dashboard.

### CODE REVIEW

**Comment by craymichael** [comment]
Let's schedule a time to talk over this after we hear from the web team.

**Comment by anurag0511** [comment]
Hey zach,

Yeah, we can talk tomorrow. I had few more changes in the data retriever
and fit bexar which was causing trouble for states. I will add that to the
PR also.

Thanks,
Anurag

File: Dashboard/get_state_data.py [hunk 1]
```diff
@@ -1,18 +1,30 @@
...
+print (paramdict['country'])
 # Retrieve the data and save to the csv
 df = retrieve_data.conflate_data(paramdict, verbose=1)
-df.to_csv("formatted_all_data.csv")
-
+print (len(df[df.Cases>0]), len(df))
+# df.to_csv("formatted_all_data.csv")
+print (df.tail(10))
 print("Finished generating data ...")
+SCRIPT_DIR = os.path.abspath(os.path.join(ROOT_DIR,'scripts'))
```
Comment (anurag0511) [inline Dashboard/get_state_data.py:23]: remove

File: scripts/forecast.py [hunk 2]
```diff
@@ -58,8 +58,12 @@ def process_data(params, df):
     :return: prev_cases - the number of cases on the day preceding day0
     """
     # Rid NaNs
-    df.dropna(inplace=True, subset=MOBILITY_KEYS)
-    df.reset_index(inplace=True)
+    print (df.Population[0],df.shape, "the total poulation in the df")
```
Comment (anurag0511) [inline scripts/forecast.py:61]: remove

**Pull request closed without merging**